packages/x-date-pickers/src/DateCalendar/DayCalendar.tsx

```diff
@@ -71,8 +71,8 @@ export interface ExportedDayCalendarProps<TDate> extends ExportedPickersDayProps
    */
   displayWeekNumber?: boolean;
   /**
-   * Calendar will show more weeks in order to match this value.
-   * Put it to 6 for having fix number of week in Gregorian calendars
+   * The calendar will show as many weeks as needed after the end of the current month to match this value.
+   * Put it to 6 to have a fix number of weeks in Gregorian calendars
```

Comment:
https://english.stackexchange.com/questions/418093/the-number-of-plural-or-singular-noun#:~:text=%22The%20number%20of%22%20should%20always,number%20of%20families%20has%20changed.

Reply:
```suggestion
   * Put it to 6 to have a fixed number of weeks in Gregorian calendars
```